### ISSUE TITLE
Feature: Replace Image Markdown with Live Image Preview

### ISSUE THREAD

**Issue by sharoonamjid** [pinkary-project/pinkary.com#717]
Currently, when an image is uploaded, the text area displays a lengthy markdown string instead of a visual preview. I propose replacing this markdown text with an actual preview of the image directly within the text area, enhancing usability by allowing users to see the image immediately rather than previewing it below the text area.

**Comment by CamKem** [comment]
I don't think we would replace the markdown, because we need to allow the user to place the image, however we could do something similar to Github & Laracasts, where there is a tab to preview how the post will look when it is sent. This could grow over time into a markdown editor style of (like what I'm typing in now)

**Comment by masihiyat** [comment]
I understand the need to keep Markdown for more control, but I believe a WYSIWYG text editor would be a more user-friendly option for most users. It allows for easy text formatting and also provides a way to insert images directly.

**Comment by sharoonamjid** [comment]
I understand the need to keep Markdown for more control, but I believe a WYSIWYG text editor would be a more user-friendly option for most users. It allows for easy text formatting and also provides a simple way to insert images directly.
